### PR TITLE
All the Docs!

### PR DESCRIPTION
**Pull request by zeke**
This PR implements the `/docs/all` route which displays everything on a single page. This PR also localizes more of the content and improves docs index styles.

### CODE REVIEW

File: styles/_basecoat-page-headers.scss [hunk 1]
```diff
@@ -58,26 +58,4 @@
...
-
-.minitron-title {
-  margin: 0;
-  color: inherit;
-  font-size: 20px;
-  font-weight: 300;
-  letter-spacing: 1px;
-  -webkit-font-smoothing: subpixel-antialiased;
-}
+}
```
Comment (codebytere) [inline styles/_basecoat-page-headers.scss]: think u need a newline here :)

File: styles/_docs.scss [hunk 2]
```diff
@@ -128,3 +128,8 @@
 
   width: 100%;
 }
+
+.docs p a,
+.docs li a {
+  text-decoration: underline;
+}
```
Comment (codebytere) [inline styles/_docs.scss]: here too

File: styles/_basecoat-page-headers.scss (hunk 1 above)
Comment (codebytere) [reply styles/_basecoat-page-headers.scss]: unless your atom config & our linter don't care about that in scss files 🤔

**Comment by zeke** [comment]
>  you'll also need to delete that stuff from the route

Cool. Will do. We should make a habit of adding tests with every PR (as appropriate) so we'll know if we ever step on each others' toes.

**Comment by codebytere** [comment]
sounds good to me! sorry about that :)

**Pull request merged**